README.md

```diff
@@ -44,9 +44,18 @@ To initialize the repository don't forget to execute `git submodule update --ini
 3) Press *BUILD* > *Build Solution* or *Rebuild Solution*.
 * __Linux & Mac OSX__:
 If you want to build with LLVM, then LLVM 3.6.2 is required.
-`cd rpcs3 && cmake CMakeLists.txt && make && cd ../` then run with `cd bin && ./rpcs3`.
```

Comment:
Hm, I think this build instruction was incorrect. You are expected to run CMake and make in project root directory, not root/rpcs3.